<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminGenerateReports</class>
 <widget class="QDialog" name="adminGenerateReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>925</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Airline System - Generate Reports</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f8f9fa;
    font-family: 'Segoe UI', Arial, sans-serif;
}
QTabWidget {
    border: none;
}
QTabBar::tab {
    background: #e0e0e0;
    color: #2c3e50;
    padding: 8px 16px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    margin-right: 2px;
    font-weight: 500;
}
QTabBar::tab:selected {
    background: white;
    border-bottom: 2px solid #3498db;
    color: #3498db;
}
QTabBar::tab:hover {
    background: #f0f0f0;
}
QGroupBox {
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    margin: 15px;
    padding: 15px;
    background-color: white;
    font-size: 14px;
    color: #2c3e50;
}
QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    padding: 10px 16px;
    border-radius: 6px;
    margin: 8px;
    font-size: 14px;
    font-weight: 500;
    min-width: 180px;
}
QPushButton:hover {
    background-color: #2980b9;
}
QPushButton:pressed {
    background-color: #2472a4;
}
QLabel#headerLabel {
    font-size: 24px;
    font-weight: 600;
    color: #000000;
    qproperty-alignment: AlignCenter;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="styleSheet">
      <string notr="true">margin-top: 15px; margin-bottom: 15px;</string>
     </property>
     <property name="text">
      <string>Report Generation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="reportsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="operationalReportsTab">
      <attribute name="title">
       <string>ðŸ“Š Operational Reports</string>
      </attribute>
      <layout class="QVBoxLayout" name="operationalReportsGroupLayout">
       <item>
        <widget class="QLabel" name="operationalReportLabel">
         <property name="text">
          <string>Enter Month and Year (MM-YYYY):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="operationalMonthYearEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="generateOperationalBtn">
         <property name="text">
          <string>Generate Operational Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="operationalReportText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="maintenanceReportsTab">
      <attribute name="title">
       <string>ðŸ”§ Maintenance Reports</string>
      </attribute>
      <layout class="QVBoxLayout" name="maintenanceReportsGroupLayout">
       <item>
        <widget class="QLabel" name="maintenanceReportLabel">
         <property name="text">
          <string>Enter Month and Year (MM-YYYY):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maintenanceMonthYearEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="generateMaintenanceBtn">
         <property name="text">
          <string>Generate Maintenance Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="maintenanceReportText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="userActivityReportsTab">
      <attribute name="title">
       <string>ðŸ‘¥ User Activity Reports</string>
      </attribute>
      <layout class="QVBoxLayout" name="userActivityReportsGroupLayout">
       <item>
        <widget class="QLabel" name="userActivityReportLabel">
         <property name="text">
          <string>Enter Month and Year (MM-YYYY):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userActivityMonthYearEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="generateUserActivityBtn">
         <property name="text">
          <string>Generate User Activity Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="userActivityReportText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backToMainMenuBtn">
     <property name="text">
      <string>Back to Main Menu</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backToMainMenuBtn</sender>
   <signal>clicked()</signal>
   <receiver>adminGenerateReports</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
